src/options/options.js

```diff
@@ -20,3 +20,5 @@ RandomTips.init(tips).then(() => {
     RandomTips.setContext("options");
     RandomTips.showRandomTipIfWanted();
 });
+
+document.getElementById("popupShowContextMenu").checked = true;
```

Comment:
Nope, this is not how we do it in this project. See [this answer](https://github.com/rugk/offline-qr-code/issues/157#issuecomment-508475513) and [this doc](https://github.com/TinyWebEx/AddonSettings#default-values) (also consider to read the whole one). You have to define the default setting in `DefaultSettings.js`.